layouts/partials/job/single_header.html

```diff
@@ -3,8 +3,10 @@
     <div class="grid grid-cols-12 gap-8 w-full items-center">
         <div class="col-span-9 text-white">
           <div class="flex items-center mb-7">
-            <img class="job-icon-sm mr-4 filter drop-shadow-lg-{{ .role }}" src='{{ print "/theme-assets/jobs/" (lower .role) "/" (lower .job) "/" (lower .job) ".svg" }}' />
-            <div class="card-title filter drop-shadow-lg-{{ .role }}">{{ .jobTitle }}</div> 
+            <a href="/jobs/{{ lower .role }}/{{ lower .jobTitle }}" class="flex flex-row" >
```

Comment:
```suggestion
            <a href="/jobs/{{ lower .role }}/{{ lower .jobTitle }}/" class="flex flex-row" >
```

Needs a trailing slash (I did the same thing in #148)

Reply:
thanks! i'll fix it up!